pkg/k8s/network_policy.go

```diff
@@ -42,6 +52,12 @@ func GetPolicyLabelsv1(np *networkingv1.NetworkPolicy) labels.LabelArray {
 }
 
 func parseNetworkPolicyPeer(namespace string, peer *networkingv1.NetworkPolicyPeer) *api.EndpointSelector {
+
+	// TODO add unit test (GH-3080).
+	if peer == nil {
+		return nil
```

Comment:
Why some functions return an error on nil arguments, others print a warning and others simply return `nil`?

Reply:
What is the desired convention? As long as we are consistent, I'm OK with whatever.